docs/content/stable/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -367,9 +432,7 @@ serviceEndpoints:
       tcp-ysql-port: "5433"
 ```
 
-For services without an explicitly defined scope in Helm overrides, the default service scope (Namespaced) is used, provided you set the **Default service scope for K8s universe** Global runtime configuration option (config key `yb.universe.default_service_scope_for_k8s`) to true. The configuration flag defines the default service scope for the universe if the scope is not explicitly defined in the service overrides.
-
-Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/). Note that only a Super Admin user can modify Global runtime configuration settings, and you cannot modify this service scope during universe creation.
+For services without an explicitly defined scope in Helm overrides, the default service scope (Namespaced) is used.
```

Comment:
This is still controlled by a runtime flag. There haven't been any changes in code to remove that runtime flag.